languages/purescript/reference/guides/Differences-from-Haskell.md

```diff
@@ -0,0 +1,315 @@
+## Evaluation strategy
```

Comment:
Some of these copied files could instead be linked to. They should attribute their sources anyway, right?